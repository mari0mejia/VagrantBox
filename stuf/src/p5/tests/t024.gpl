int i=99;
on akey 
{
  i=0;
  //Empty initializer. Should be fine
  for(; i<5; i++)
    print("Counting: " + i);
}

on akey{print("");}

on akey 
{
  //Empty post-loop statement. Should be fine
  for(i=0; i<5;)
  {
    print("Counting: " + i);
    i++;
  }
}

on akey{print("");}

on akey 
{
  i=0;
  //Empty initializer, empty post-loop statement. Should be fine
  for(;i<5;)
  {
    print("Counting: " + i);
    i++;
  }
}
