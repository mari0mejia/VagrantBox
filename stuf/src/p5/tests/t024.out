gpl.cpp::main()
  input file(tests/t024.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int i = 99
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[7]: Counting: 0
print[7]: Counting: 1
print[7]: Counting: 2
print[7]: Counting: 3
print[7]: Counting: 4
print[10]: 
print[17]: Counting: 0
print[17]: Counting: 1
print[17]: Counting: 2
print[17]: Counting: 3
print[17]: Counting: 4
print[22]: 
print[30]: Counting: 0
print[30]: Counting: 1
print[30]: Counting: 2
print[30]: Counting: 3
print[30]: Counting: 4

User quit program. Printing the symbol table.
int i = 5
